### DISCUSSION TITLE
Does Nuxt Has A plan of going Full Stack?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#23976] [Questions] [answered]
I have been reading Remix, what I like about Remix is you can do frontend and backend, and I also like MVC's like Laravel. You can work with Front and backend as well. Does Nuxt have a plan going full stack? currently, I am using the server to build minor API and some minor stuff only But I am not really sure if I should use it fully to work as a backend.

Or is Nuxt already a full-stack?

**Comment by AloisSeckar** [comment, marked as answer]
Yes, it is already full-stack. In Nuxt 3 you can implement server-side logic easily:
https://nuxt.com/docs/getting-started/server

It allows you to expose API enpoints easily just by having files with `defineEventHandler` functions inside `/server` directory and you can put whaever backend logic you want. Their code is executed on the server, not in the client's browser.
